pyaedt/modules/SetupTemplates.py

```diff
@@ -1,2380 +1,1500 @@
-    ("Solution Initialization - Turbulent Dissipation Rate", "1m2_per_s3"),
-    ("Solution Initialization - Specific Dissipation Rate", "1diss_per_s"),
-    ("Convergence Criteria - Flow", "0.001"),
-    ("Convergence Criteria - Energy", "1e-07"),
-    ("Convergence Criteria - Turbulent Kinetic Energy", "0.001"),
-    ("Convergence Criteria - Turbulent Dissipation Rate", "0.001"),
-    ("Convergence Criteria - Specific Dissipation Rate", "0.001"),
-    ("Convergence Criteria - Discrete Ordinates", "1e-06"),
-    ("IsEnabled", False),
-    ("Radiation Model", "Off"),
-    ("Under-relaxation - Pressure", "0.7"),
-    ("Under-relaxation - Momentum", "0.3"),
-    ("Under-relaxation - Temperature", "1"),
-    ("Under-relaxation - Turbulent Kinetic Energy", "0.8"),
-    ("Under-relaxation - Turbulent Dissipation Rate", "0.8"),
-    ("Under-relaxation - Specific Dissipation Rate", "0.8"),
-    ("Discretization Scheme - Pressure", "Standard"),
-    ("Discretization Scheme - Momentum", "First"),
-    ("Discretization Scheme - Temperature", "Second"),
-    ("Secondary Gradient", False),
-    ("Discretization Scheme - Turbulent Kinetic Energy", "First"),
-    ("Discretization Scheme - Turbulent Dissipation Rate", "First"),
-    ("Discretization Scheme - Specific Dissipation Rate", "First"),
-    ("Discretization Scheme - Discrete Ordinates", "First"),
-    ("Linear Solver Type - Pressure", "V"),
-    ("Linear Solver Type - Momentum", "flex"),
-    ("Linear Solver Type - Temperature", "F"),
-    ("Linear Solver Type - Turbulent Kinetic Energy", "flex"),
-    ("Linear Solver Type - Turbulent Dissipation Rate", "flex"),
-    ("Linear Solver Type - Specific Dissipation Rate", "flex"),
-    ("Linear Solver Termination Criterion - Pressure", "0.1"),
-    ("Linear Solver Termination Criterion - Momentum", "0.1"),
-    ("Linear Solver Termination Criterion - Temperature", "0.1"),
-    ("Linear Solver Termination Criterion - Turbulent Kinetic Energy", "0.1"),
-    ("Linear Solver Termination Criterion - Turbulent Dissipation  Rate", "0.1"),
-    ("Linear Solver Termination Criterion - Specific Dissipation Rate", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Pressure", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Momentum", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Temperature", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Turbulent Kinetic Energy", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Turbulent Dissipation Rate", "0.1"),
-    ("Linear Solver Residual Reduction Tolerance - Specific Dissipation Rate", "0.1"),
-    ("Linear Solver Stabilization - Pressure", "None"),
-    ("Linear Solver Stabilization - Temperature", "None"),
-    ("Frozen Flow Simulation", False),
-    ("Sequential Solve of Flow and Energy Equations", False),
-    ("Convergence Criteria - Max Iterations", 100),
-]
+SteadyTemperatureAndFlow = OrderedDict(
+    {
+        "Enabled": True,
```

Comment:
This item has a mix of colons and dashes. It should be consistent. Colons are preferred.